docs/faqs.rst

```diff
@@ -0,0 +1,87 @@
+=============================
+FAQs
+=============================
+
+Below are a list of frequently asked questions (and answers) for users new to Planemo.
+
+What is Planemo?
+=============================
+
+Planemo is a command-line application which is primarily used for developing Galaxy and CWL tools,
+workflows and training materials. It's also used to deploying the tools and workflows once developed
+(tools to the Galaxy ToolShed; workflows to the Intergalactic Workflow Commission) and executing
+Galaxy-based analyses via the command-line, if you prefer or need to use that rather than Galaxy's
+graphical interface.
+
+Is Planemo the right tool for me?
+=================================
+
+Yes, if you are comfortable using the command-line and one of the following is true:
+  - You want to use a piece of scientific software not currently available in Galaxy and therefore need to wrap it yourself.
+  - You want to develop CWL tools or workflows.
```

Comment:
Should we add here Galaxy tools as well?

Reply:
Or maybe add a new bullet point about developing Galaxy workflows?

Reply:
> Should we add here Galaxy tools as well?

That's meant to be the first point.